core/domain/suggestion_registry.py

```diff
@@ -64,6 +64,8 @@ class BaseSuggestion(python_utils.OBJECT):
         language_code: str|None. The ISO 639-1 code used to query suggestions
             by language, or None if the suggestion type is not queryable by
             language.
+        edited_by_reviewer: Boolean. The flag to identify whether the
```

Comment:
Boolean is not a python type. Please use whatever `type(...)` gives you. Ditto elsewhere.

Also, you can start the description with "Whether"; you don't need the first 4 words.

Reply:
Done